Sources/Plasma/Apps/plClient/Mac-Cocoa/PLSPatcher.h

```diff
@@ -50,7 +50,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)patcher:(PLSPatcher*)patcher beganDownloadOfFile:(NSString*)file;
 - (void)patcher:(PLSPatcher*)patcher updatedProgress:(NSString*)progressMessage withBytes:(NSUInteger)bytes outOf:(uint64_t)totalBytes;
-- (void)patcherCompleted:(PLSPatcher*)patcher;
+- (void)patcherCompleted:(PLSPatcher*)patcher didSelfPatch:(BOOL)selfPatched;
 - (void)patcherCompletedWithError:(PLSPatcher*)patcher error:(NSError*)error;
```

Comment:
I realize this doesn't _actually_ matter, but per Apple API selector naming patterns, shouldn't this be `patcherCompleted:withError:`?  😝

Reply:
With should only be used if there is another word to connect to. So something like initWithError is ok because "with" is the glue between init and error. For a freestanding argument you wouldn't add the with, such as `stringWithContentsOfFile: encoding: error:`.